src/richie/apps/courses/templates/courses/cms/organization_detail.html

```diff
@@ -20,15 +20,15 @@
                 {% endget_placeholder_plugins %}
                 {% blockplugin plugins.0 %}
                     <img
-                        src="{% thumbnail instance.picture 320x100 upscale subject_location=instance.picture.subject_location %}"
+                        src="{% thumbnail instance.picture 320x100 crop upscale subject_location=instance.picture.subject_location %}"
                         srcset="
-                            {% thumbnail instance.picture 640x200 upscale subject_location=instance.picture.subject_location %} 600w,
-                            {% thumbnail instance.picture 1000x200 upscale subject_location=instance.picture.subject_location %} 1000w
-                            {% if instance.picture.width >= 1500 %},{% thumbnail instance.picture 1500x300 upscale subject_location=instance.picture.subject_location %} 1500w{% endif %}
-                            {% if instance.picture.width >= 2000 %},{% thumbnail instance.picture 2000x400 upscale subject_location=instance.picture.subject_location %} 2000w{% endif %}
-                            {% if instance.picture.width >= 2500 %},{% thumbnail instance.picture 2500x500 upscale subject_location=instance.picture.subject_location %} 2500w{% endif %}
+                            {% thumbnail instance.picture 640x200 crop upscale subject_location=instance.picture.subject_location %} 600w,
```

Comment:
can you explain more in details in the commit what was the problem. Cropping the image will have a different effect and I think it might be problematic for logos...

Reply:
This is something I have to investigate thats why I let the `WIP` label